testsuite/tests/typing-objects/pr6123_bad.compilers.reference

```diff
@@ -1,8 +1,7 @@
-
-#                                             Characters 253-257:
+Characters 279-283:
```

Comment:
In most cases, the character ranges were not changed by the commit. This is the first case I see where the location changes, is it normal/expected?

Reply:
Presumably this is because the error is in the first structure item of the file, and the newly added (* TEST .. *) comment at the front shift the locations in the first structure item.